source/functions.rst

```diff
@@ -1,191 +1,186 @@
 Functions
 *********
 
-You can think of functions as actions, verbs, or commands 
-and you can think of parameters as adverbs: 'run, quickly'
+A functions is some code which has been given a name and which can be used
+wherever you need it in your program. When we use a function we say we are
+"calling" it.
 
-Functions are special objects that contain code.
+Some examples of functions we've already used are ``print``, ``int`` and
+``type``.
 
-When you call them, using special syntax `()`, you execute the
-code they contain.
+    >>> print("hello") hello
```

Comment:
Maybe give an example of all the three functions that have been used? I think repetitive is good.